src/derivatives/05-diffusion-derivatives.md

```diff
@@ -517,38 +506,37 @@ another.
                                  [ 128.8 ] ],
         "SphericalHarmonicDegree": 0,
         "Tissue": "Gray matter"
+        },
+        "csf":
+        {
+            XXX
```

Comment:
Should change PR to draft given that it is explicitly not completed.

Reply:
Oops - yeah - changed to WIP in PR title to indicate we aren't done here.